subgraph-js/src/federation-atlas.ts

```diff
@@ -0,0 +1,62 @@
+import { Atlas, Schema, gql } from "@apollo/core-schema";
+
+export const SUBGRAPH_BASE = Schema.basic(
+  gql `${'builtin:subgraph-base'}
+    @link(url: "https://specs.apollo.dev/federation/v1.0", as: "",
+          import: "@key @requires @provides @external")
+    @link(url: "https://specs.apollo.dev/tag/v0.1")
+  `)
+
+export const ATLAS = Atlas.fromSchemas(
+  Schema.basic(gql `${'builtin/tag/v0.1'}
+  @id(url: "https://specs.apollo.dev/tag/v0.1")
+  directive @tag(name: String!)
+    repeatable on FIELD_DEFINITION | INTERFACE | OBJECT | UNION
+  `),
+  Schema.basic(gql `${'builtin/federation/v1.0.graphql'}
+  @id(url: "https://specs.apollo.dev/federation/v1.0")
+
+  """
+  federation 1.0 key directive
+  """
+  directive @key(fields: FieldSet!) repeatable on OBJECT | INTERFACE
+  directive @requires(fields: FieldSet!) on FIELD_DEFINITION
+  directive @provides(fields: FieldSet!) on FIELD_DEFINITION
+  directive @external on OBJECT | FIELD_DEFINITION
+
+  scalar FieldSet
+  `),
+
+  Schema.basic(gql `${'builtin/federation/v2.0.graphql'}
+  @id(url: "https://specs.apollo.dev/federation/v2.0")
+
+  """
+  federation 2.0 key directive
```

Comment:
It's missing:
- `@inaccessible`: which has been recently added (#1484).
- `@extend`: see comment above.
- `@tag`? Subgraphs should be able to `import` `@tag` from the fed2 feature. Does that work if `@tag` is not defined here?

Reply:
fixed